examples/.tailcallrc.graphql

```diff
@@ -300,6 +300,66 @@ directive @cache(
   maxAge: Int!
 ) on FIELD_DEFINITION
 
+directive @expr(body: ExprBody!) on FIELD_DEFINITION
+
+input ExprBody {
+  if: ExprIf
+  http: ExprHttp
+  grpc: ExprGrpc
+  const: JSON
+  graphQL: ExprGraphQL
+}
+
+input ExprIf {
+  cond: ExprBody!
+  then: ExprBody!
+  else: ExprBody!
+}
+
+input ExprHttp {
+  baseURL: String
+  path: String!
+  method: Method
+  body: String
+  headers: [KeyValue]
+  query: [KeyValue]
+  groupBy: [String!]
+}
+
+input ExprGrpc {
+  service: String!
+  method: String!
+  body: String
+  headers: [KeyValue]
+  protoPath: String
+  groupBy: [String!]
+}
+
+input ExprGraphQL {
```

Comment:
we need to add comments (`""" some doc """`) to these also

Reply:
There is work on auto-generating this file in #895, including doc comments

Reply:
Whoever will work on #895, it'll copy the comments that will have there and pass it to the `.rs` equivalent stuff. Since #895 does not include write new docs, it's fair that we add these now.